staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -560,7 +560,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		if err != nil {
 			return nil, fmt.Errorf("failed to create field manager: %v", err)
 		}
-		reqScope.FieldManager = fm
+		reqScope.FieldManager = fieldmanager.NewLazyFieldManager(fm, a.group.Creater, fqKindToRegister)
```

Comment:
Why don't we create the lazyfieldmanager at the same time that we create the fieldmanager?